docs/source/concepts/ledger-model/ledger-daml.rst

```diff
@@ -33,6 +33,7 @@ Intuitively, the allowed actions are:
 
    #. The actors match the controllers of the choice.
       That is, the controllers define the :ref:`required authorizers <da-ledgers-required-authorizers>` of the choice.
+   #. The choice observers match the observers annotated in the choice.
```

Comment:
@oggy-  I added this line here in the "Daml: Defining Contract Models Compactly" page. (Not going into details on the DAML syntax for choice observers here, since that's a separate part of the docs.)